OpenTrace/Monitoring/Symptoms/SymptomsViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SymptomsViewController.swift
+//  OpenTrace
+//
+//  Created by Sam Dods on 15/04/2020.
+//  Copyright © 2020 OpenTrace. All rights reserved.
+//
+
+import UIKit
+
+class SymptomsViewController: UIViewController {
+
+    convenience init() {
+        self.init(nibName: String(describing: SymptomsViewController.self), bundle: Bundle(for: SymptomsViewController.self))
```

Comment:
I like this even from the Main Bundle!!!!!!

Reply:
haha, autopilot :)